spec/index.html

```diff
@@ -515,20 +515,99 @@ <h2>Basic Concepts</h2>
     </p>
 
     <p class="ednote">FIXME.</p>
+
+    <section class="informative" id="convenience-context">
+      <h3>Convenience Context</h3>
+      <p>As JSON-LD keywords cannot be represented in YAML without being quoted,
+        publishers may find it useful to declare term aliases for these
+        keywords.</p>
+
+      <p>
+        For convenience, a <dfn>convenience context</dfn> is available
+        at `https://www.w3.org/ns/yaml-ld/v1`, which pre-defines these aliases:
+      </p>
+
+      <pre data-transform="updateExample">
+        <!--
+        {
+          "@context": {
+            "$always": "@always",
+            "$base": "@base",
+            "$container": "@container",
+            "$direction": "@direction",
+            "$embed": "@embed",
+            "$explicit": "@explicit",
+            "$graph": "@graph",
+            "$id": "@id",
+            "$import": "@import",
+            "$included": "@included",
+            "$index": "@index",
+            "$json": "@json",
+            "$language": "@language",
+            "$list": "@list",
+            "$nest": "@nest",
+            "$never": "@never",
+            "$none": "@none",
+            "$null": "@null",
+            "$omitDefault": "@omitDefault",
+            "$once": "@once",
+            "$prefix": "@prefix",
+            "$propagate": "@propagate",
+            "$protected": "@protected",
+            "$requireAll": "@requireAll",
+            "$reverse": "@reverse",
+            "$set": "@set",
+            "$type": "@type",
+            "$value": "@value",
+            "$version": "@version",
+            "$vocab": "@vocab"
+          }
+        }
+        -->
+      </pre>
+
+      <p>The <a>convenience context</a> contains an alias to every JSON-LD keyword which the JSON-LD 1.1
+        specification permits aliasing &mdash; which means all the keywords except <code>@context</code>. The reserved `@` character is
+        replaced by `$`, which is not reserved and therefore does not require quoting. Consider
+        <a href="#example-example-yaml-ld-document-without-convenience-context"></a>
+         reformatted using the <a>convenience context</a>:</p>
+
+    <p>This allows YAML-LD to be created with minimal use of <a>map</a> quoted keys (Note that the `@context` keyword cannot be aliased):</p>
```

Comment:
The "note" is redundant to a couple paragraphs above, and the "minimal use" sentence doesn't quite flow from the paragraph immediately preceding...

Reply:
Please consider updated wording of the paragraph.

Reply:
Fixed in 76ceeb26a770d522e64b9173158242a11d4a1ea0.